src/tools/sssctl/sssctl_cache.c

```diff
@@ -31,10 +31,10 @@
 #define NOT_FOUND_MSG(obj) _(obj " %s is not present in cache.\n")
 
 #define SSSCTL_CACHE_NAME   {_("Name"), SYSDB_NAME, attr_name_fn}
-#define SSSCTL_CACHE_CREATE {_("Cache entry creation date"), SYSDB_CREATE_TIME, attr_time}
-#define SSSCTL_CACHE_UPDATE {_("Cache entry last update time"), SYSDB_LAST_UPDATE, attr_time}
-#define SSSCTL_CACHE_EXPIRE {_("Cache entry expiration time"), SYSDB_CACHE_EXPIRE, attr_expire}
-#define SSSCTL_CACHE_IFP    {_("Cached in InfoPipe"), SYSDB_IFP_CACHED, attr_yesno}
+#define SSSCTL_CACHE_CREATE {_("Cache entry creation date"), SYSDB_CREATE_TIME, attr_time_fn}
+#define SSSCTL_CACHE_UPDATE {_("Cache entry last update time"), SYSDB_LAST_UPDATE, attr_time_fn}
+#define SSSCTL_CACHE_EXPIRE {_("Cache entry expiration time"), SYSDB_CACHE_EXPIRE, attr_expire_fn}
+#define SSSCTL_CACHE_IFP    {_("Cached in InfoPipe"), SYSDB_IFP_CACHED, attr_yesno_fn}
 #define SSSCTL_CACHE_NULL   {NULL, NULL, NULL}
```

Comment:
Suffix "_fn" in name of function looks weird to me.
I would suggest either rename function (get_attr_name ...) to have more suitable name
or rename local variables which shadow functions.